ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh

```diff
@@ -103,20 +103,12 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
     fi
 
     # Build a source code
-    cd build
-    cmake -DCMAKE_INSTALL_PREFIX=${NNST_ROOT} -DCMAKE_INSTALL_LIBDIR=lib ..
+    meson --werror --prefix=${NNST_ROOT} --libdir=lib --bindir=bin --includedir=include -Dinstall-example=true build
```

Comment:
Could you share us the experimental result  that you tested with this PR #448 in your own Ubuntu 16.04?

Reply:
Let's merge this PR after doing the aging test enough.